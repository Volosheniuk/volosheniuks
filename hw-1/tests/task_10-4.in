Q  12 36512 -0.29
