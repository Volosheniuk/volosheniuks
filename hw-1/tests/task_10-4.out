Q 12 36512 -0.290000
